src/features/Courses/CoursesTable/_test_/columns.test.jsx

```diff
@@ -3,7 +3,7 @@ import { columns } from 'features/Courses/CoursesTable/columns';
 describe('columns', () => {
   test('returns an array of columns with correct properties', () => {
     expect(columns).toBeInstanceOf(Array);
-    expect(columns).toHaveLength(5);
+    expect(columns).toHaveLength(6);
```

Comment:
I would suggest to add a new test case to ensure the open can be open/closed and  if the actions are called.